index.md

```diff
@@ -4,67 +4,171 @@ This is some welcome text.
 
 ## Pylint errors {#pylint}
 
-### R0101: Too many nested blocks {#R0101}
 
-This error occurs when you have more than three nested blocks in your code.
-This type of nesting is a sign that your function is too complex,
-and should be broken down using helper functions.
+### E0601: Used before assignment {#E0601}
 
-Note: the nested block doesn't doesn't include function or class definition
-as a block, so the example below is considered to have *four* nested blocks,
-not five.
+This error occurs when you are using a variable before it's assignment.
 
 ```python
-def my_f(num):
-    if num > 0:
-        if num > 3:
-            if num > 10:
-                if num < 50:
-                    print(num)
-    else:
-        print('done')
+print(a)   # Error on this line
+a = 1
+```
+
+
+### E0602: Undefined variable {#E0602}
+
+This error occurs when you are using a variable that has not been defined.
+
+```python
+var1 = 1
+var2 = None
+
+print(var1)
+print(var2)   # Error on this line
+```
+
+
+### E0611: No name in module {#E0611}
+
+This error occurs when you are trying to access a variable from an imported
+module, but that variable name could not be found in that referenced module.
+
+```python
+""" mod1.py """   # inside of mod1 module.
+
+__all__ = ['a', 'b']
+
+a = 1
+b = 2 #   Note: There is no variable called c.
+
+""" mod2.py """   # inside of mod1 module.
+
+from mod1 import *
+
+print(mod1.a)
+print(mod1.b)
+print(mod1.c) # But c is used here, which has caused an error.
 ```
 
-### R0102: Simplifiable if statement {#R0102}
 
-This error occurs when you have an if statement that can be simplified
-simply by using the value of the condition, rather than putting in extra
-`True` and `False` literals.
+### E0632: Unbalanced tuple unpacking {E0632}
+
+This error occurs when you have a possibly unbalance unpacking assignment
```

Comment:
This sentence has too many undefined terms. Your second setence is fine - remove the first sentence and the "Note".